### ISSUE TITLE
GT HD maps available — why still use pretrained segmentation?

### ISSUE THREAD

**Issue by ShenZheng2000** [boschresearch/PseudoMapTrainer#6]
Hi, authors! thanks for your excellent work! After reading your paper, I have a question below. 

Since datasets like nuScenes and Argoverse already provide GT HD map annotations in vector format, why do you still use a pretrained segmentation model to fill the semantic classes instead of directly using the GT vector maps?

**Comment by christianll9** [comment]
Hi @ShenZheng2000, thanks for your interest!

The main contribution of our work was to show that it is *possible* to train with pseudo labels instead of the ground truth HD map. Here, the GT map was essential to quantitatively evaluate our pseudo labels (see Table 1). This should demonstrate the usefulness of pseudo labels for datasets without GT. Hope that helps!

**Comment by ShenZheng2000** [comment]
@christianll9 Thanks so much!
